### DISCUSSION TITLE
Sort by State

### DISCUSSION THREAD

**Discussion by v0lli** [brutella/hkknx-public#349] [Ideas]
Is it possible to make the State-column also sortable?
And maybe even a nested sorting by room, kind, state?

thanks

<img width="1346" alt="Screenshot 2024-11-04 at 11 20 51" src="https://github.com/user-attachments/assets/dce19aed-ee0c-43dc-b7cd-051fb49ba3e3">

**Comment by brutella** [comment]
I'm hesitant to add this kind of feature because the table would re-sort automatically. This may be problematic if you want to interact with a device and just before clicking on a switch, the table re-sorts and you mistakenly click on a different switch.
